src/content/blog/en/zundamon-lt-project.md

```
@@ -0,0 +1,137 @@
+title: "[Marp×VOICEVOX×VTubeStudio] Zundamon Gave an LT Presentation"
+description: "Creative use of technology! An experimental project building an automated LT presentation system using Zundamon with a combination of Marp, VOICEVOX, and VTubeStudio."
+pubDate: 2024-01-30
+author: "Terisuke"
+category: "lab"
+tags: ["Marp", "VOICEVOX", "VTubeStudio", "自動化", "創造的プロジェクト"]
+# image:
+#   url: "/images/blog/zundamon-project.avif"
+#   alt: "ずんだもんLTプロジェクト"
+lang: "en"
+---
+# Marp × VOICEVOX × VTubeStudio:  Having Zundamon Deliver a Lightning Talk
```

markdownlint-cli2 (0.17.2)

Multiple top-level headings in the same document
null

(MD025, single-title, single-h1)